### ISSUE TITLE
session object recreated everytime

### ISSUE THREAD

**Issue by chaojinn** [mewebstudio/captcha#82]
Hi I followed the instruction to create a test route
```
Route::any('captcha-test', function()
    {
        if (Request::getMethod() == 'POST')
        {
            $rules = ['captcha' => 'required|captcha'];
            $validator = Validator::make(Input::all(), $rules);
            if ($validator->fails())
            {
                echo '<p style="color: #ff0000;">Incorrect!</p>';
            }
            else
            {
                echo '<p style="color: #00ff30;">Matched :)</p>';
            }
        }

        $form = '<form method="post" action="captcha-test">';
        $form .= '<input type="hidden" name="_token" value="' . csrf_token() . '">';
        $form .= '<p>' . captcha_img() . '</p>';
        $form .= '<p><input type="text" name="captcha"></p>';
        $form .= '<p><button type="submit" name="check">Check</button></p>';
        $form .= '</form>';
        return $form;
    });
```
But the validation seems to be failing every time. After a little bit digging. I found in 
Captcha.php -> public function check($value) function 
The check 
```
if ( ! $this->session->has('captcha'))
{
return false;
}
```
is always failing
It seems when the function generate is called, it will call __construct once
and before check is called, __construct is called again.
So it is always trying to get the captcha from a different session.
I am not quite familiar with Laravel itself, my test is based on version Laravel Framework version 5.2.31

**Comment by fzhan** [comment]
I think you meant that captcha always refresh itself regardless if the application decide to keep the last captcha?

I believe this is the intention of the code owner for this version, at least last version this wasn't the case.

**Comment by chaojinn** [comment]
Not really, it seems session->has('captcha') is always false , so the validation of code always failed. This can be easily replicated by  adding 
Route::any('captcha-test', function()
    {
        if (Request::getMethod() == 'POST')
        {
            $rules = ['captcha' => 'required|captcha'];
            $validator = Validator::make(Input::all(), $rules);
            if ($validator->fails())
            {
                echo '<p style="color: #ff0000;">Incorrect!</p>';
            }
            else
            {
                echo '<p style="color: #00ff30;">Matched :)</p>';
            }
        }

        $form = '<form method="post" action="captcha-test">';
        $form .= '<input type="hidden" name="_token" value="' . csrf_token() . '">';
        $form .= '<p>' . captcha_img() . '</p>';
        $form .= '<p><input type="text" name="captcha"></p>';
        $form .= '<p><button type="submit" name="check">Check</button></p>';
        $form .= '</form>';
        return $form;
    });
in routes.php. 
I am testing with  Laravel Framework version 5.2.31 and the latest version of mewebstudio/captcha

**Comment by ailmcm** [comment]
Good day, in readme.md in vendor folder (differs from actual here):

> for Laravel 5.2+
> Add `\Illuminate\Session\Middleware\StartSession::class` to `$middleware` in `Kernel.php`
> 
they meant [app/Http/Kernel.php]
actually it helped me

**Comment by yulinyige** [comment]
@ailmcm I work for me. you save my life, thank you.
